lib/elements/satin_column.py

```diff
@@ -1464,9 +1472,21 @@ def inset_short_stitches_sawtooth(self, pairs):
     def _get_inset_point(self, point1, point2, distance_fraction):
         return point1 * (1 - distance_fraction) + point2 * distance_fraction
 
+    def add_running_stitches(self, start_stitch, end_stitch, patch):
+        # when max_stitch_length is set, the satin column may be quite wide annd  the jumps
+        # between underlays or from underlay to final satin are better turned into paths
+        # to avoid long jumps or  trims
```

Comment:
```suggestion
        # When max_stitch_length is set, the satin column may be quite wide and the jumps
        # between underlays or from underlay to final satin are should be turned into
        # running stitch to avoid long jumps or unexpected trims.
```